src/components/interaction-designer/ResourceViewer.vue

```diff
@@ -4,7 +4,7 @@
     <div class="resource-viewer-contents">
       <resource-viewer-block
         v-for="block in activeFlow?.blocks ?? []"
-        v-show="visibleBlocks.includes(block.uuid)"
+        v-show="visibleBlocks === null || visibleBlocks.includes(block.uuid)"
```

Comment:
I'm proposing here to always display those block if the visibleBlocks is null, which would allow us to make a test on that edge case scenario.